spec/factories/vaccination_records.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: vaccination_records
+#
+#  id                 :bigint           not null, primary key
+#  administered_at    :date
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  patient_session_id :bigint           not null
+#
+# Indexes
+#
+#  index_vaccination_records_on_patient_session_id  (patient_session_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (patient_session_id => patient_sessions.id)
+#
+FactoryBot.define do
+  factory :vaccination_record do
+    patient_session { nil }
+    administered_at { "2023-06-09" }
```

Comment:
I expect in the (near) future we may want to generate these when we create a session, maybe with a cascading approach. This works for now.